solutions/commons/utils/authorize/getRedirectToClientRedirectUri.ts

```diff
@@ -5,8 +5,9 @@ export const getRedirectToClientRedirectUri = (
   error?: (typeof authorizeErrors)[keyof typeof authorizeErrors],
   state?: string,
   code?: string,
+  redirectUriIsRelative = false,
 ) => {
-  const url = new URL(redirectUri);
+  const url = new URL(redirectUri, "http://localhost");
```

Comment:
If the `redirectUri` is an invalid URL it will fallback to `http://localhost` - would it be better to catch Invalid URL error instead and redirect to the error page if the client's redirect is invalid? Obviously we validate the redirect_uri in schema and check it against the client registry so this should never occur in reality but if it did getting the user to go to http://localhost would be weird imho.